_includes/footer.html

```diff
@@ -32,7 +32,7 @@
     <div class="site-info small">
             <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><i class="fa fa-cc fa-2x"></i></a>
       <p>Except where otherwise noted content created by the <a xmlns:cc="http://creativecommons.org/ns#" href="https://www.verapdf.org" property="cc:attributionName" rel="cc:attributionURL"> veraPDF Consortium</a> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.</p>
-      <p>Copyright &copy; 2015 - 2023 veraPDF consortium</p>
+      <p>Copyright &copy; 2015 - 2024 veraPDF consortium</p>
```

Comment:
What if we use this instead: `<script>document.write(new Date().getFullYear());</script>`

Reply:
That would be better, will take a look.